plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleTablePhysical.java

```diff
@@ -77,6 +79,14 @@ protected OracleTablePhysical(
         this.partitionCache = partitioned ? new PartitionCache() : null;
     }
 
+    protected OracleTablePhysical(@NotNull OracleSchema schema, @NotNull ResultSet dbResult, @NotNull String name) {
+        // Table partition
+        super(schema, name);
+        this.rowCount = JDBCUtils.safeGetLong(dbResult, "NUM_ROWS");
```

Comment:
Please avoid copy-paste. Move NUM_ROWS and TABLESPACE_NAME read to a separate method

Reply:
implemented